### ISSUE TITLE
Code example is terribly wrong

### ISSUE THREAD

**Issue by complainer-MSFT** [microsoftgraph/microsoft-graph-docs#20491]
What in God's name is going on here?  This example on your page is crazy.  You have backticks where you want backslashes, and you have an en-dash next to a hyphen.  Did u OCR ur code example??>!

```
az resource create --resource-group myRG --name myGraphAppBilling --resource-type Microsoft.GraphServices/accounts --properties  "{`"appId\`": `"myAppGUID`"}" --latest-include-preview --location Global –-subscription mySubscriptionGUID
```

Specifically, this part is bogus, right?  Those backticks are visually kinda similar to backslashes, but close is not good enough:

```
"{`"appId\`": `"myAppGUID`"}"
```

And this part is bogus too.  Look closely at those hyphens!  Well, one hyphen and one en-dash...

```
–-subscription mySubscriptionGUID
```

On a page where people are trying to figure out how to pay you more money...

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: b356f4b8-50b3-7ff6-8c3f-a5210b027c7c
* Version Independent ID: b356f4b8-50b3-7ff6-8c3f-a5210b027c7c
* Content: [Enable metered Microsoft 365 APIs and services - Microsoft Graph](https://learn.microsoft.com/en-us/graph/metered-api-setup)
* Content Source: [concepts/metered-api-setup.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/concepts/metered-api-setup.md)
* Product: **non-product-specific**
* Technology: **microsoft-graph**
* GitHub Login: @JeremyKelley
* Microsoft Alias: **MSGraphDocsVteam**

**Comment by complainer-MSFT** [comment]
Here's the line you want instead, with fixed backslashes and hyphens:

```
az resource create --resource-group myRG --name myGraphAppBilling --resource-type Microsoft.GraphServices/accounts --properties  "{\"appId\": \"myAppGUID\"}" --latest-include-preview --location Global --subscription mySubscriptionGUID
```

**Comment by wbrown0389** [comment]
@complainer-MSFT this corrected formatting didn't work for me. I keep getting the below error, with my app ID appropriately replaced.

`unrecognized arguments: appId\:\myAppGUIDHere\ }`

Did you ever get this to work?

**Comment by simongeary** [comment]
I'm also having this problem even after replacing the backticks, same error. Anyone have a solution?

unrecognized arguments: appId\:\foo\ }

**Comment by simongeary** [comment]
I got this working in the end by (1) replacing the backticks with backslashes and (2) using Azure Cloud Shell instead of my locally installed copy of AZ CLI.

My local CLI and Cloud Shell were both version 2.46.0 so I am unsure on the difference in behaviour.

My working command was.

az resource create --resource-group RG-Name --name Billing-Name --resource-type Microsoft.GraphServices/accounts --properties  "{\"appId\": \"cef3c8c6-20bc-2e8f-bfb3-7f549ca6e2f7\"}" --location Global --subscription 3b0d9281-1740-4a66-bbc8-86b1720ee7bc

**Comment by wbrown0389** [comment]
Response I got back from support notes that using local PowerShell requires you to escape the backticks using \ before the backtick (I can't show it here because markdown removes the backslash). This is apparently due to Python (Azure CLI) and PowerShell mixing escape characters. Reference [here](https://github.com/Azure/azure-cli/blob/dev/doc/quoting-issues-with-powershell.md)

**Comment by complainer-MSFT** [comment]
Yeah, I was using Azure Cloud Shell, and escaping with regular backslashes as @simongeary did above worked successfully for me.  

Dumb question: how about just using single quotes instead of having to escape anything?

`az resource create --resource-group RG-Name --name Billing-Name --resource-type Microsoft.GraphServices/accounts --properties  '{"appId": "cef3c8c6-20bc-2e8f-bfb3-7f549ca6e2f7"}' --location Global --subscription 3b0d9281-1740-4a66-bbc8-86b1720ee7bc`

**Comment by white-tim** [comment]
Here is the powershell command I used:
```
Install-Module az.resources
Import-Module az.resources
Connect-AzAccount
Register-AzResourceProvider -ProviderNamespace Microsoft.GraphServices
New-AzResource -ResourceName myRg -ResourceGroupName myAppBilling -ResourceType Microsoft.GraphServices/accounts -Properties @{"appId"="myAPPGUID"} -Location Global
```

And here is the Cloud Shell

`az resource create --resource-group RESOURCEGROUPNAME --name RESOURCENAME --resource-type Microsoft.GraphServices/accounts --properties  "{ \"appId\":\"00000000-0000-0000-0000-000000000000\" }" --location Global --subscription 00000000-0000-0000-0000-000000000000`
